directdemod/server/conf.json

```diff
@@ -1 +1 @@
-{"update_rate": 20, "counter": 4, "1": "1", "2": "2019-06-26_08:07:33_2019-06-26_08:07:53", "3": "2019-06-26_08:07:53_2019-06-26_08:08:13"}
\ No newline at end of file
+{"update_rate": 20, "counter": 4, "1": "1", "2": "2019-07-27_10:55:02_2019-07-27_10:55:22", "3": "2019-07-29_09:16:12_2019-07-29_09:16:32"}
```

Comment:
I understand the "update_rate" parameter. What do the rest do?

Reply:
The counter is the parameter for a number of intervals (number of days). Next, are the pairs: day id and the corresponding time interval. So when it is chosen, the corresponding time label will be shown also.